integrations/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -632,6 +632,19 @@ void showLog() throws NessieConflictException, NessieNotFoundException {
         .containsExactlyElementsOf(resultList);
   }
 
+  @Test
+  void showLogAt() throws NessieConflictException, NessieNotFoundException {
+    List<SparkCommitLogEntry> resultList = createBranchCommitAndReturnLog();
+
+    // here we are skipping commit time as its variable
```

Comment:
Can you add a test using the timestamp as in `throwWhenUseShowReferencesAtTimestampWithoutTimeZone()` ?

Reply:
done

Reply:
The test code below uses a hash value for the `AT` clause (which is a good test). Please add another test using a timestamp with the `AT` clause.

Reply:
done.